go/worker/common/p2p/peermgmt.go

```diff
@@ -92,7 +92,11 @@ func (mgr *PeerManager) isPeerAuthorized(id core.PeerID) bool {
 	mgr.RLock()
 	defer mgr.RUnlock()
 
-	return mgr.peers[id] != nil
+	// TODO: if we're txnscheduler anyone can publish, unless we would use a
+	// separate gossipsub network for that.
+	// TODO: add a settable 'allowAll' flag, that we can toggle when we
```

Comment:
Not sure if this is a good idea as it could result in a flaky gossipsub topology.

Reply:
I wouldn't use `allowAll`.